### ISSUE TITLE
Y-Axis formatting missing in explorer pages

### ISSUE THREAD

**Issue by makeavish** [SigNoz/signoz#2934]
For graphs in explorer pages, Y-Axis formatting is required as charts like duration are not readable in nanoseconds.
Eg:
<img width="1235" alt="Screenshot 2023-06-19 at 17 58 18" src="https://github.com/SigNoz/signoz/assets/9512100/123e1fa4-aa7e-46bd-aa0e-ac4c5eaf9c9e">

**Comment by pranay01** [comment]
So, is the suggestion to show the y-axis unit dynamically based on the value?
i.e instead of 105 mn nano second, show 0.105 **s**?
and if if is 105,000 nano second - it should show 0.105 **ms**

**Comment by makeavish** [comment]
> So, is the suggestion to show the y-axis unit dynamically based on the value? i.e instead of 105 mn nano second, show 0.105 **s**? and if if is 105,000 nano second - it should show 0.105 **ms**

We can show dynamically or we can allow user to manually select y-axis format similar to current dashboards query editor.

**Comment by palashgdev** [comment]
@SigNoz/backend is it possible to be part of the composite query

**Comment by srikanthccv** [comment]
@palashgdev Yes, It will be part of the composite query (Not there yet and should be added). You will use the same when you redirect to alerts(https://github.com/SigNoz/signoz/issues/3016) or dashboards.

**Comment by pranay01** [comment]
@kaushikmacharla07 have a look at possible UX here

**Comment by kaushikmacharla07** [comment]
**Design :** https://www.figma.com/file/i2CsZFl27dhmER6gnx08wj/Integ.-QueryBuilder%2B3SigNalCorrelation?type=design&node-id=3735%3A19808&mode=design&t=SVjio4lKsPhM8V8R-1

**Comment by palashgdev** [comment]
> **Design :** https://www.figma.com/file/i2CsZFl27dhmER6gnx08wj/Integ.-QueryBuilder%2B3SigNalCorrelation?type=design&node-id=3735%3A19808&mode=design&t=SVjio4lKsPhM8V8R-1

we are introducing the unit is composite query level how this will reflect in another view? apart from time series

**Comment by kaushikmacharla07** [comment]
You will be seeing different unit in the table in other views.

**Comment by srikanthccv** [comment]
This is present now
